tests/workflows/dissociation_curve/test_workchain_diss_curve.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=redefined-outer-name
+"""Tests for the :mod:`aiida_common_workflows.workflows.dissociation` module."""
+import pytest
+
+from aiida import orm
+from aiida.engine import WorkChain
+from aiida.plugins import WorkflowFactory
+
+from aiida_common_workflows.plugins import get_workflow_entry_point_names
+from aiida_common_workflows.workflows import dissociation
+from aiida_common_workflows.workflows.relax.workchain import CommonRelaxWorkChain
+
+
+@pytest.fixture
+def ctx():
+    """Return the context for a port validator."""
+    return None
+
```

Comment:
I don't think there is a test for `validate_inputs` which I think is quite important. It should test that it raises if neither `distances` nor all of the other three are defined, and what should happen if both sets are defined. As said in another comment, since the `distance_max` etc. define defaults, they will _always_ be passed, even if `distances` is explicitly defined and `distances` should then take the precedenc

Reply:
I'm noticing now that the test is missing also for the eos... And the not tested `validate_inputs` of eos contains a bug! I will fix it here if you don't mind!